docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+---
+title: Serial Vault User Guide
+table_of_contents: true
+---
+
+# Serial Vault user guide
+
+The Serial Vault is a Canonical-hosted web service that is required to use a
+[Secure Brand Store][secure-brand-store]. The Serial Vault provides :
+
+ - Signed [serial assertions][serial-assertions]  to authenticated systems of
+   a given brand/model, which then use it to access a secure Brand Store.
+ - Signed [system-user assertions][system-user-assertions], which allow
+   creating a system user account from a USB drive on an unmanaged Core system.
+   For additional information, see [here][here].
+
+After Canonical creates a Serial Vault account for you, you can login here:
+[https://serial-vault-partners-admin.canonical.com/]
+
+Serial Vault tasks include:
+
+ - Generating and uploading new serial keys.
+ - Creating new models and associating them with a key and your brand.
+ - Reviewing serial signing logs.
+ - Optionally creating system user assertions.
+
+
+!!! Note:
+          The Brand SSO account is used for all brand activities, including
+          generating the serial key, as explained here.
+
+Please contact Canonical to request a Serial Vault user account. The account
+must be associated with the Brand SSO account of your Brand Store. Canonical
+will need the following:
+
+ - The Username of the brand SSO account from [https://login.ubuntu.com/][https://login.ubuntu.com/] 
```

Comment:
These all needs a period at the end to be consistent with other bulleted lists in this doc.

Reply:
Would be good to explain how to contact Canonical